File: python/github/get/credentials/src/formula/formula.py
#!/usr/bin/python3
import json

class Github:
	username = 'username'
	token = 'token'
	email = 'email'

def Run(username, token, email):
    #create object
    github = Github()
    github.username = username
    github.token = token
    github.email = email

    #convert to JSON string
    github_json = json.dumps(github.__dict__)

    #print json string
    print(github_json)

File: go/github/get/credentials/src/pkg/formula/formula.go
package formula

import (
	"bytes"
	"encoding/json"
	"log"
)

type Formula struct {
	Username string
	Token    string
}

func (h Formula) Run() {
	response, err := json.Marshal(h)
	if err != nil {
		log.Printf("Error", h)
	}

	var prettyJSON bytes.Buffer
	error := json.Indent(&prettyJSON, response, "", "\t")
	if error != nil {
		log.Println("JSON parse error:", error)
	}

	log.Println("Github Credentials:", string(prettyJSON.Bytes()))
}
